packages/catalog-realm/fields/number.gts

```diff
@@ -0,0 +1,2581 @@
+          outline-offset: 2px;
+        }
+        .dual-range__input:focus-visible::-moz-range-thumb {
+          outline: var(--boxel-outline);
+          outline-offset: 2px;
+        }
+        .dual-range__input:disabled::-webkit-slider-thumb,
+        .dual-range__input:disabled::-moz-range-thumb {
+          border-color: var(--boxel-300);
+          background: var(--boxel-light-500);
+          box-shadow: none;
+          opacity: 0.5;
+        }
+        .dual-range__inputs {
+          display: grid;
+          grid-template-columns: 1fr auto 1fr;
+          gap: var(--boxel-sp-xs);
+          align-items: center;
+        }
+        .dual-range__input-control {
+          width: 100%;
+        }
+        .dual-range__separator {
+          font-size: var(--boxel-font-size-sm);
+          font-weight: var(--boxel-font-weight-normal);
+          color: var(--muted-foreground, var(--boxel-450));
+        }
+        .dual-range__bounds {
+          display: flex;
+          justify-content: space-between;
+          font-size: var(--boxel-font-size-xs);
+          font-weight: var(--boxel-font-weight-normal);
+          letter-spacing: var(--boxel-lsp-xs);
+          color: var(--muted-foreground, var(--boxel-500));
+        }
+      }
+    </style>
+  </template>
+}
+
+class DualRangeSliderField extends FieldDef {
+  static displayName = 'Dual Range Slider';
+  @field min = contains(NumberField);
+  @field max = contains(NumberField);
+  static edit = DualRangeSliderFieldEdit;
+  static atom = class DualRangeSliderAtom extends Component<
+    typeof DualRangeSliderField
+  > {
+    get config(): DualRangeSliderConfig {
+      return readConfig(this.args.configuration, {
+        min: 0,
+        max: 1000,
+        step: 10,
+        allowNegative: false,
+        decimals: 0,
+        currency: '$',
+        showInputs: true,
+      });
+    }
+
+    get minValue(): number | null {
+      let raw = this.args.model.min;
+      if (typeof raw === 'number') {
+        return clampValue(raw, this.config);
+      }
+      return null;
+    }
+
+    get maxValue(): number | null {
+      let raw = this.args.model.max;
+      if (typeof raw === 'number') {
+        return clampValue(raw, this.config);
+      }
+      return null;
+    }
+
+    get displayValue() {
+      let minLabel =
+        this.minValue != null
+          ? formatRangeValue(this.minValue, this.config)
+          : 'Any';
+      let maxLabel =
+        this.maxValue != null
+          ? formatRangeValue(this.maxValue, this.config)
+          : 'Any';
+
+      if (minLabel === 'Any' && maxLabel === 'Any') {
+        return 'Any range';
+      }
+      return `${minLabel} - ${maxLabel}`;
+    }
+
+    <template>
+      <span class='dual-range-atom'>{{this.displayValue}}</span>
+      <style scoped>
+        .dual-range-atom {
+          font-weight: var(--boxel-font-weight-semibold);
+          color: var(--primary, var(--boxel-dark));
+        }
+      </style>
```

Comment:
let's remove the styling here and let the user style it how ever they want. This looks inconsistent with the rest of the embedded views.